web/modules/custom/asset_distribution/asset_distribution.module

```diff
@@ -138,21 +140,42 @@ function asset_distribution_rdf_entity_view(array &$build, EntityInterface $enti
  * Implements hook_ENTITY_TYPE_presave().
  */
 function asset_distribution_rdf_entity_presave(RdfInterface $distribution) {
-  if ($distribution->bundle() !== 'asset_distribution') {
+  global $base_url;
+  if ($distribution->bundle() != 'asset_distribution') {
     return;
   }
+  $file_urls = [];
+  $file_size = 0;
+  $access_urls = $distribution->get('field_ad_access_url')->getValue();
+
+  // Get all file urls.
+  /** @var \Drupal\field\FieldConfigInterface $file_field */
+  foreach ($distribution->get('field_ad_file') as $file_field) {
+    $file = File::load($file_field->get('target_id')->getValue());
+    if ($file) {
+      $file_urls[] = $file->url();
+    }
+
+    $file_size += $file->getSize();
```

Comment:
The sum of the file sizes looks a bit weird, but it should only be 1 file anyway.